source/OptimaJet.DWKit.StarterApplication/Repositories/ControllerRepository.cs

```diff
@@ -4,23 +4,33 @@
 using JsonApiDotNetCore.Services;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Logging;
+using OptimaJet.DWKit.StarterApplication.Models;
 
 namespace OptimaJet.DWKit.StarterApplication.Repositories
 {
-    public class ControllerRepository<TEntity> : DefaultEntityRepository<TEntity>
+    public class BaseRepository<TEntity> : DefaultEntityRepository<TEntity>
         where TEntity : class, IIdentifiable<int>
     {
         protected readonly DbSet<TEntity> dbSet;
+        protected readonly CurrentUserRepository currentUserRepository;
         protected readonly DbContext dbContext;
 
-        public ControllerRepository(
+        public BaseRepository(
```

Comment:
Had to rename this, because repositories aren't used by controllers. :-\

Reply:
Rename the file too?

Reply:
silly IDE.

Reply:
done